features/admin.console-settings.v1/components/console-administrators/administrators-list/administrators-list.tsx

```diff
@@ -531,7 +559,7 @@ const AdministratorsList: FunctionComponent<AdministratorsListProps> = (
                     defaultListItemLimit={ defaultListItemLimit }
                     administrators={ administrators }
                     onUserEdit={ handleUserEdit }
-                    onUserDelete={ handleUserDelete }
+                    onUserDelete={ isCentralDeploymentEnabled ? handleGuestUserDelete : handleUserDelete }
```

Comment:
Let's add the reason for this conditional in the PR description.

Reply:
Added a description